### ISSUE TITLE
Error: channel closed

### ISSUE THREAD

**Issue by uchamb** [avocode/avocode#340]
App v.2.3.0-ddf81ed
Source UID: 93727
Please try to describe how did it happen.

-----
```
Error: channel closed
    at ChildProcess.target.send (child_process.js:396:26)
    at Task.module.exports.Task.send (/home/ucha/Avocode/resources/app.asar/src/task.js:125:27)
    at e._send (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/downloader.js:1:5895)
    at e._addRequest (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/downloader.js:1:3910)
    at e.downloadFiles (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/downloader.js:1:1981)
    at e.downloadFiles (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/api.js:1:6381)
    at t.module.exports.t.downloadThumbnails (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/assets-downloader.js:1:1032)
    at e._downloadThumbnails (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/user.js:1:15437)
    at e._updateDesignThumbnail (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/user.js:1:19279)
    at /home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/user.js:1:18508
    at /home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/api.js:1:801
    at Request._callback (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/lib/api.js:1:3053)
    at Request.self.callback (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/node_modules/request/request.js:197:22)
    at emitTwo (events.js:92:20)
    at Request.emit (events.js:169:7)
    at Request.<anonymous> (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core/node_modules/request/request.js:1050:14)
    at emitOne (events.js:82:20)
    at Request.emit (events.js:166:7)
    at IncomingMessage.<anonymous> (/home/ucha/Avocode/resources/app.asar/node_modules/@avocode/core
```

**Comment by rychlis** [comment]
Please try to describe how did it happen.

**Comment by petrbrzek** [comment]
That happend when child process terminates. We use Atom Task for syncing.

**Comment by rychlis** [comment]
We are working on a fix but this issue is duplicate of #388 , I am only keeping 388 open now
